config/initializers/barong.rb

```diff
@@ -36,6 +36,31 @@
 
 kstore = Barong::KeyStore.new(pkey)
 
+Barong::App.define do |config|
```

Comment:
there is no sense to duplicate `Barong::App.define do |config|` in the same file
Move it to corresponding initializer, if you think it makes sense or move it in the already created `define` module